src/helpers/constants/validation.schemas.js

```diff
@@ -95,5 +95,15 @@ export default {
     rating: number
       .valid(numbers.ONE, numbers.TWO, numbers.THREE, numbers.FOUR, numbers.FIVE)
       .required()
+  }),
+  reportArticle: Joi.object().keys({
+    reason: number.min(1).integer().required()
+  }),
+  updateArticle: Joi.object().keys({
+    title: Joi.string().allow(''),
```

Comment:
@MCFrank16 I was actually asking here if we want to have articles with an empty title/body/category? Open to hearing what you think.

Reply:
no, we don't, but for I to test the Existing data, I need to send empty data so that it can take up the existing data and have no changes in the database, that is why I allowed null in validation.